test.js

```diff
@@ -805,24 +901,24 @@ describe('General error handling', function () {
     expect(response.json()).toEqual({
       statusCode: 401,
       error: 'Unauthorized',
-      message: 'Invalid token.'
+      message: 'The token is malformed.'
     })
   })
 
-  it('should complain if the JWT token has an unsupported algorithm', async function () {
-    const response = await server.inject({
-      method: 'GET',
-      url: '/verify',
-      headers: { Authorization: `Bearer ${tokens.unsupportedAlgorithm}` }
-    })
+  // it('should complain if the JWT token has an unsupported algorithm', async function () {
```

Comment:
I don't know if it's ok, just reminding you that this test is commented

Reply:
Good catch. Thanks.